scripts/download_cams_pm25.py

```
@@ -0,0 +1,267 @@
+                                  f"{sensor.parameter.value},"
+                                  f"{float(m.value)},"
+                                  f"{sensor.unit.value},"
+                                  f"{'forecast' if use_forecast else 'reanalysis'}\n")
+                            measurement_count += 1
+                    
+                total_measurements += measurement_count
+                
+                if measurement_count > 0:
+                    logger.info(f"  {sensor.parameter.value}: {measurement_count} measurements")
+        
+        logger.info(f"Download complete!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)